src/azure-cli-core/azure/cli/core/_identity.py

```diff
@@ -63,6 +63,8 @@ def __init__(self, authority=None, tenant_id=None, client_id=None, scopes=None,
         self._cred_cache = kwargs.pop('cred_cache', None)
         self.allow_unencrypted = kwargs.pop('allow_unencrypted', True)
         self.scopes = scopes
+        if self.scopes and not isinstance(self.scopes, (list, tuple)):
+            self.scopes = (self.scopes,)
```

Comment:
Use `list` as MSAL uses `list` as the type:

```suggestion
            self.scopes = [self.scopes]
```